SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -69,6 +70,7 @@ bool USpatialNetDriver::InitBase(bool bInitAsClient, FNetworkNotify* InNotify, c
 
 	bConnectAsClient = bInitAsClient;
 	bAuthoritativeDestruction = true;
+	bQueueOpsUntilReady = !bInitAsClient;
```

Comment:
Nit: would also want other opinions but I think maybe semantically it more intuitive to have the bool be `IsReady` rather than `QueueUntilReady`. Because then false means you are queuing and true means you are ready. Don't have strong opinions about this.

Reply:
I'd prefer a more descriptive name, like IsReadyToCallBeginPlay or similar if you'd prefer the logic inversion.  Don't feel all that strongly about this change either

Reply:
i'll invert the logic and call it bIsReadyToStart